source/appendix_errors.rst

```diff
@@ -162,25 +162,9 @@ Executing `l` results in::
      10  	print(f())
     [EOF]
 
-Step through each line of code keeping. 
+Step through each line of code keeping.
 
 Ensure you explore the two frames when you enter the f functions' frame.
 
 .. tip::
     We have used a python code visualiser in a similar way.
-
-
-
-koans & `pdb`
-=============
-
-`pdb` is a great tool to understand code. Here we will apply it to our koans.
```

Comment:
I think the pdb section has enough information to get someone up and running the koans section didn't really add much to that other than a chance to test it out on your own stuff